src/common/low_precision_transformations/tests/multiply_transformation.cpp

```diff
@@ -82,916 +193,560 @@ TEST_P(MultiplyTransformation, CompareFunctions) {
     ASSERT_TRUE(LayerTransformation::allNamesAreUnique(actualFunction)) << "Not all names are unique";
 }
 
-const std::vector<ov::element::Type> precisions = {
+const std::vector<ov::element::Type> model_precisions = {
     ov::element::f32,
     ov::element::f16
 };
 
-const std::vector<MultiplyTransformationTestValues> multiplyTransformationTestValues = {
-    // U8
+const std::vector<std::pair<ov::element::Type, ov::element::Type>> input_precisions = {
+    { ov::element::u8, ov::element::u8 },
+    { ov::element::i8, ov::element::i8 },
+    { ov::element::u8, ov::element::i8 },
+    { ov::element::i8, ov::element::u8 },
+    { ov::element::f32, ov::element::f32 },
+    { ov::element::f16, ov::element::f16 },
+};
+
+namespace broadcast_no {
+const std::vector<std::pair<PartialShape, PartialShape>> input_shapes = {
+    {{ 1, 3, 8, 16 }, { 1, 3, 8, 16 }},
+    {{ 1, 3, 8, 16 }, { 1, 3, 1, 1 }},
+    {{ 1, 3, 1, 1 }, { 1, 3, 8, 16 }},
```

Comment:
`broadcast_no` name doesn't correspond to the test-cases with broadcasted input shapes. Can we rename it?

Reply:
done